README.md

```diff
@@ -69,8 +79,18 @@ The following values are returned in `results.csv`:
 
 #### Mail sending
 
-* `MX Record` - If an MX record was found that contains at least a single mail server.
+* `MX Record` - If an MX record was found that contains at least a
+  single mail server.
 * `Mail Servers` - The list of hosts found in the MX record.
+* `Mail Server Ports Tested` - A list of the ports tested for SMTP and
+  STARTTLS support.
+* `Mail Server Is Listening` - If the mail servers are actually
+  listening on the tested ports.
+* `Mail Server Supports SMTP` - If the mail servers support SMTP.
```

Comment:
I propose that the field `Mail Server Supports SMTP` become Boolean (true if at least one domain on the MX record "speaks" SMTP), with the current `Mail Server Supports SMTP` becoming `Mail Server Supports SMTP Results`.

Reply:
Maybe it makes sense to rename both fields from `Mail Server Supports...` to `Domain Supports...`, since it's semantically odd for a `*Mail Server* Supports SMTP` to be false (for a "mail server" to not offer SMTP).

Reply:
:+1:

Reply:
@jsf9k last thing: let's put the `...Supports` Boolean *before* the `...Supports Results` field, which is helpful from a human-using-the-tool-output use case.